couchable/core.py

```diff
@@ -78,10 +89,11 @@ def importstr(module_str, from_=None):
         return getattr(module, from_)
     return module
 
+
 def typestr(type_):
     if not isinstance(type_, type):
         type_ = type(type_)
-    if type_.__name__ in __builtins__:
+    if hasattr(builtins, type_.__name__) or type_ == type(None):
```

Comment:
`__builtins__` is a implementation detail of CPython and should never be used. This is the portable and correct way to check for builtin types with an added check for `None`, since this a global singleton.